app/views/teacher_interface/application_forms/show/_declined.html.erb

```diff
@@ -9,11 +9,15 @@
     <h4 class="govuk-heading-s"><%= title %></h4>
   <% end %>
 
-  <ul class="govuk-list govuk-list--bullet">
-    <% reasons.each do |reason| %>
-      <li><%= simple_format reason %></li>
-    <% end %>
-  </ul>
+  <% if reasons.count == 1 %>
+    <%= simple_format reasons.first %>
```

Comment:
would this single reason be showing a bullet point? The commit suggests you'd want a single bullet point but I don't think the code would use a bullet point here.